tests/platform_tests/conftest.py

```diff
@@ -141,6 +143,21 @@ def get_kexec_time(duthost, messages, result):
 
 def analyze_log_file(duthost, messages, result, offset_from_kexec):
     service_restart_times = dict()
+    derived_patterns = OTHER_PATTERNS.get("COMMON")
+    service_patterns = dict()
+    # get platform specific regexes
+    if is_broadcom_device(duthost):
+        derived_patterns.update(OTHER_PATTERNS.get("BRCM"))
+    elif is_mellanox_device(duthost):
+        derived_patterns.update(OTHER_PATTERNS.get("MLNX"))
+    # get image specific regexes
+    if "20191130" in duthost.os_version:
+        derived_patterns.update(OTHER_PATTERNS.get("201911"))
+        service_patterns.update(SERVICE_PATTERNS.get("201911"))
+    else:
+        derived_patterns.update(OTHER_PATTERNS.get("LATEST"))
+        service_patterns.update(SERVICE_PATTERNS.get("LATEST"))
```

Comment:
If the version is 201811, it will use latest. Wondering if it should be closer to 201911?

Reply:
201811 should ideally have its own regexes. However, for 201811 images we do not fully support pytest, and still run Ansible based tests.